docs/securing_fledge.rst

```diff
@@ -349,23 +349,20 @@ Default ca certificate is available inside $FLEDGE_DATA/etc/certs and named as c
 
 Below are the steps to create custom certificate along with existing fledge based ca signed for auth certificates.
 
-a) Create a new certificate for username. Let say **test**
+a) To create a user with the name **test** (case sensitive), please note that only an admin has the permission to do so. The cURL commands are provided below.
```

Comment:
To create a user with a unique and case-insensitive username (e.g., uname1), you can follow these steps:

a) Using cURL

Create User
Create Authentication Certificate 

b) later via GUI

....

Note: _You need administrator privileges to perform this action._

Reply:
We can also extend the create user API to allow generate auth cert with some flag in payload? To avoid 2 steps process?

Reply:
Extension is okay but it is good to have separate endpoint and do not want to add all stuff in one go. It would also admin will see what action required for the user.

Reply:
Yes

Hence I meant a flag to allow as an option while creating user(name).